### PR TITLE
masurca a new recipes

### PR DESCRIPTION
**Pull request by mictadlo**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/masurca/build.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+echo `pwd`
+outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+mkdir -p $outdir
+mkdir -p $PREFIX/bin
+cd $SRC_DIR
+echo $SRC_DIR
+
+./install.sh
```
Comment (bgruening) [inline recipes/masurca/build.sh]: You probably need to pass here the directory where you want to install your binaries ... $PREFIX/bin.

Comment (mictadlo) [inline recipes/masurca/build.sh]: It looks like the `install.sh` script does not take any parameters. Please find below:
```
#!/bin/sh

set -xe

ROOT=`pwd`
[ -z "$DEST" ] && DEST="$ROOT"

###################
# Check for gmake #
###################
mkdir -p dist-bin
PATH=$PATH:$ROOT/dist-bin
ln -sf $(which make) $ROOT/dist-bin/gmake
ln -sf $ROOT/PkgConfig.pm $ROOT/dist-bin/pkg-config

export NUM_THREADS=`grep -c '^processor' /proc/cpuinfo 2>/dev/null || sysctl -n hw.ncpu 2>/dev/null || echo 1`;
BINDIR=$DEST/bin
LIBDIR=$DEST/lib
export PKG_CONFIG_PATH=$LIBDIR/pkgconfig:$PKG_CONFIG_PATH
(cd global-1 && ./configure --prefix=$DEST --bindir=$BINDIR --libdir=$LIBDIR && make -j $NUM_THREADS install-special)

echo "creating sr_config_example.txt with correct PATHS"
$BINDIR/masurca -g sr_config_example.txt
```

What else could I try?

Best wishes,

Michal

Comment (bgruening) [inline recipes/masurca/build.sh]: You can set $DEST.
`export DEST=$PREFIX`

**Comment by mictadlo** [comment]
I changed it, but still, it failed. Any what could I miss?

**Comment by bgruening** [comment]
https://travis-ci.org/bioconda/bioconda-recipes/jobs/182563139#L555
```
/opt/conda/conda-bld/work/conda_build.sh: line 5: ./install.sh: No such file or directory
```

**Comment by bgruening** [comment]
Maybe this file is not executable?

**Comment by mictadlo** [comment]
I found a linuxbrew masurca [package](https://github.com/Homebrew/homebrew-science/blob/master/masurca.rb), but I do not how to convert it to conda.

**Comment by mictadlo** [comment]
No error anymore, but still failed tobjilf

**Pull request closed without merging**